packages/common-all/src/util/regex.ts

```diff
@@ -15,3 +15,9 @@ export function isWebUri(uri: string): boolean {
   if (scheme === "http" || scheme === "https") return true;
   return false;
 }
+
+export function isCommandUri(uri: string): boolean {
```

Comment:
can we add a quick docstring for this util method? 

Also - this might be better if it has 'VSCode' somewhere in the function name (it's VSCode Command Uri right)?

Reply:
instead of `vscode`, would use `code` (since it can also refer to `vscodium`)

Reply:
> instead of vscode, would use code (since it can also refer to vscodium)

I think we can stick to `vscode` since every other places we just refer it to vscode. (i.e. `VSCodeUtils`)

I propose that we keep it consistent and later discuss if we want to change the naming scheme, and then bulk refactor everything that refers to it as `vscode`